src/components/dev-dot-content/mdx-components/mdx-headings/index.tsx

```diff
@@ -12,27 +12,31 @@ const HEADING_LEVELS_TO_PROPS: Record<
 	{ size: HeadingProps['size']; weight: HeadingProps['weight'] }
 > = {
 	1: {
+		/**
+		 * Note: font-size is bumped to a custom "Display 600", which does
+		 * not yet exist in HDS (has been designed for Dev Dot specifically).
+		 */
 		size: 500,
 		weight: 'bold',
 	},
 	2: {
-		size: 400,
+		size: 500,
```

Comment:
Have update the type stack for MDX headings (ie all long form content) to match design specs.